composer.json

```diff
@@ -13,7 +13,9 @@
         "sparkphp/command": "self.version"
     },
     "require": {
-        "php": ">=5.5"
+        "php": ">=5.5",
+        "equip/assist": "^0.1.2",
+        "equip/data": "^2.3"
```

Comment:
> Anywhere you want an immutable value object without entity/repo stuff.

Well, yeah. I included that in the question when I asked it. This doesn't answer it. I'm looking for specifics.

> This could be really useful for notifications too.

How so? Would we not have entities or repositories for notifications?